packages/theme/src/components/number-input.ts

```diff
@@ -32,19 +33,25 @@ const baseStyle = (props: Record<string, any>) => ({
   stepper: baseStyleStepper(props),
 })
 
-function getSize(size: "sm" | "md" | "lg") {
-  const sizeStyle = Input.sizes?.[size]
+function getSize(size: "xs" | "sm" | "md" | "lg") {
+  const sizeStyle = Input.sizes[size]
 
   const radius = {
     lg: "md",
     md: "md",
     sm: "sm",
+    xs: "sm",
   }
 
+  const resolvedFontSize = Typography.fontSizes[sizeStyle.field.fontSize]
+
   return {
-    field: sizeStyle?.field,
+    field: {
+      ...sizeStyle.field,
+      verticalAlign: "top",
```

Comment:
When using NumberInput with size="xs", the stepper is slightly shifted down.
This fixes it.